File: examples/swagger-3/src/test/resources/asserts/java/retrofit/src/com/wuxp/codegen/swagger3/domain/BaseInfo.java
package com.wuxp.codegen.swagger3.domain;

import lombok.Data;
import lombok.experimental.Accessors;
import jakarta.validation.constraints.*;
import com.fasterxml.jackson.annotation.JsonFormat;
        import com.wuxp.codegen.swagger3.domain.BaseExample;

@Data
@Accessors(chain = true)
public class  BaseInfo<I,T> {

        protected I id;

        protected T data;

            /**
                *字段在java中的类型为：BaseExample
            **/
        protected BaseExample example;

}

File: examples/swagger-3/src/test/resources/asserts/typescript/typescript_feign/src/domain/BaseInfo.ts
/* tslint:disable */
/* eslint-disable */

        import {BaseExample} from "./BaseExample";
import {DefaultOrderField} from "feign-client";



export interface  BaseInfo<ID,T> {

        id?: ID;
        data?: T;
            /**
                *字段在java中的类型为：BaseExample
            **/
        example?: 'A';
}